controller/array_action/array_mediator_ds8k.py

```diff
@@ -650,10 +639,9 @@ def validate_flashcopy(self, flashcopy_id):
         api_flashcopy = self._get_flashcopy(flashcopy_id)
         return api_flashcopy.state == 'valid'
 
-    def _check_snapshot_use_status(self, snapshot_id, flashcopy_list):
-        for flashcopy in flashcopy_list:
-            logger.info("Deleting flashcopy: {}".format(flashcopy))
+    def _check_snapshot_use_status(self, snapshot_id, flashcopies):
+        for flashcopy in flashcopies:
             if flashcopy.sourcevolume == snapshot_id:
-                flashcopy_rel = self._get_flashcopy(flashcopy.id)
-                if flashcopy_rel.out_of_sync_tracks != '0':
-                    raise array_errors.SnapshotIsStillInUseError(snapshot_id, flashcopy_rel.targetvolume)
+                flashcopy_obj = self._get_flashcopy(flashcopy.id)
```

Comment:
https://github.com/IBM/ibm-block-csi-driver/pull/185#discussion_r443222200
what is the difference between `flashcopy` and `flashcopy_obj` here?

Reply:
flashcopy is small version of flashcopy in the storage, flashcopy_obj is the whole object from the storage the contain data of the copy process

Reply:
I feel this fact is not really conveyed with those current names.
consider renaming `flashcopy_obj` to something like `flashcopy_process` (everywhere relevant, i.e. including its `get` function)

Reply:
done